package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "werelogs",
   "engines": {
-    "node": ">=16"
+    "node": ">=10"
```

Comment:
if version 16 is too high should we perhaps ask for version 10 only, or may be something lower than 16 ? or lower than X where X is the version that brakes compatibility because right now running version 16 or higher seems still a valid choice.

Reply:
I think keeping it like >= 10 makes sense at the moment, we will need to come back on a later stage to this project to increase the minimum target node version when the prod. environments get upgraded to 16.
We can also try to create an easy pipeline to make sure the project compiles on 10, 12, 14, 16 and immediately notice when it breaks due to any change.
CC: @tcarmet

Reply:
now that the project got moved to actions, be my guest if you want to create a testing matrix https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-running-multiple-versions-of-nodejs

Reply:
@tcarmet for me it would be ideal